#constant for arm
uint8 DISARM_COMMAND = 0
uint8 ARM_COMMAND    = 1

#request
uint8 arm    # see constants
---
bool result
# for debugging usage
uint8 cmd_set
uint8 cmd_id
uint32 ack_data
